verify_rpms/rpm_verifier.py

```diff
@@ -155,33 +158,22 @@ def __call__(self, img: str) -> ProcessedImage:
     type=click.Path(path_type=Path),
     required=True,
 )
-@click.option(
-    "--status-path",
-    help="Path in which the status will be written to",
-    type=click.Path(path_type=Path),
-    required=True,
-)
```

Comment:
What's the reason for making this non-configurable?

Reply:
I could not find a reason to leave it configurable.
We have a workdir, we can use it for the status (and in another PR for the results)
We already doing it in the task itself:
https://github.com/redhat-appstudio/tekton-tools/blob/main/tasks/verify-signed-rpms/0.1/verify-signed-rpms.yaml#L49

Reply:
I think we should not remove something that is working and tested as part of an unrelated PR.